karrot/activities/permissions.py

```diff
@@ -16,6 +16,13 @@ def has_object_permission(self, request, view, obj):
             return obj.is_upcoming()
 
 
+class IsNotUpcoming(permissions.BasePermission):
+    message = 'The activity is not in the past.'
+
+    def has_object_permission(self, request, view, obj):
```

Comment:
Should we also allow all SAFE_METHODS here to be consistent with IsUpcoming?

Reply:
Please elaborate 😄 🛩️